cpp/src/arrow/filesystem/azurefs_test.cc

```diff
@@ -1382,6 +1386,29 @@ TEST_F(TestAzuriteFileSystem, DeleteDirContentsFailureNonexistent) {
   this->TestDeleteDirContentsFailureNonexistent();
 }
 
+TEST_F(TestAzuriteFileSystem, DeleteFileSuccessHaveFile) {
```

Comment:
Could you simplify test name?

```suggestion
TEST_F(TestAzuriteFileSystem, DeleteFileSuccess) {
```